api/queries/live-time-top-developers-by-prs-daily/template.sql

```diff
@@ -1,17 +1,24 @@
-SELECT
-        actor_id,
-        ANY_VALUE(actor_login) AS actor_login,
-        COUNT(DISTINCT CASE WHEN action = 'opened' THEN pr_or_issue_id ELSE NULL END) AS opened_prs,
-        COUNT(DISTINCT CASE WHEN action = 'closed' AND pr_merged = false THEN pr_or_issue_id ELSE NULL END) AS closed_prs,
-        COUNT(DISTINCT CASE WHEN action = 'closed' AND pr_merged = true THEN pr_or_issue_id ELSE NULL END) AS merged_prs,
-        COUNT(DISTINCT pr_or_issue_id) AS total_prs
-FROM github_events ge
-WHERE
-        type = 'PullRequestEvent'
-        AND action IN ('opened', 'closed')
-        AND created_at > DATE_SUB(NOW(), INTERVAL 1 DAY)
-        AND actor_login NOT LIKE '%[bot]'
-        AND actor_login NOT LIKE '%bot'
-GROUP BY actor_id
+WITH pr_24h AS (
+	SELECT
+                /*+ MERGE() */
+                actor_id, actor_login,
+                COUNT(DISTINCT CASE WHEN action = 'opened' THEN pr_or_issue_id ELSE NULL END) AS opened_prs,
+                COUNT(DISTINCT CASE WHEN action = 'closed' AND pr_merged = false THEN pr_or_issue_id ELSE NULL END) AS closed_prs,
+                COUNT(DISTINCT CASE WHEN action = 'closed' AND pr_merged = true THEN pr_or_issue_id ELSE NULL END) AS merged_prs
+	FROM
+                github_events ge
+	WHERE
+                type = 'PullRequestEvent'
+                AND (action = 'opened' OR action = 'closed')
+                AND created_at > DATE_SUB(NOW(), INTERVAL 1 DAY)
+                AND actor_login NOT LIKE '%[bot]'
+                AND actor_login NOT LIKE '%bot'
+	GROUP BY
+                actor_id, actor_login
+)
+SELECT 
+	actor_id, actor_login, (opened_prs + closed_prs + merged_prs) as total_prs,
```

Comment:
There may be intersection in opened_prs / closed_prs / merged_prs.

Reply:
Maybe alias should be `total_pr_events`

Reply:
Yep, it is exactly what we want.

![image](https://user-images.githubusercontent.com/26212551/190133969-69f0f0c8-0fb2-4539-87c1-3e765c9a2d8f.png)